2_form.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>My first form here</title>
+</head>
+<body>
+
+  <h1>Payment Form</h1>
+  <h4>Requierd Fields are followed by .*.</h4>
+  <p style="font-size:30px">Contact Information</p> <!--Using this to write here in large size-->
```

Comment:
Contact Information is part of the <form>

Reply:
I didn't get it ? Where are u targetting ?

Reply:
I mean part of the form. Contact info is part of the form. So put it within the form tag